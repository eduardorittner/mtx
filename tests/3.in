This is a testto see
whether new line insert works
