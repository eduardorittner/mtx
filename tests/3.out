This is a test
to see
whether new line insert works
